_posts/2023-03-09-group-equivariant-cnn-1/group-equivariant-cnn-1.Rmd

```diff
@@ -0,0 +1,148 @@
+---
+title: "Upside-down, a cat's still a cat: Evolving image recognition with Geometric Deep Learning"
+description: >
+  In this first in a series of posts on group-equivariant convolutional neural networks (GCNNs), meet the main actors -- groups -- and concepts (equivariance). With GCNNs, after a break far too long, we finally revisit the topic of Geometric Deep Learning, a principled, math-driven approach to neural networks that has consistently been rising in scope and impact.
```

Comment:
```suggestion
  In this first in a series of posts on group-equivariant convolutional neural networks (GCNNs), meet the main actors -- groups -- and concepts (equivariance). With GCNNs, we finally revisit the topic of Geometric Deep Learning, a principled, math-driven approach to neural networks that has consistently been rising in scope and impact.
```

Reply:
Ah okay, that helps. I added that "after the fact", since the "finally" seemed to need some explanation ... But if you don't think so, all the better!